src/iosource/af_packet/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+if (${CMAKE_SYSTEM_NAME} MATCHES Linux)
+    set(ZEEK_HAVE_AF_PACKET yes CACHE INTERNAL "")
+
+    include(CheckSymbolExists)
+
+    check_symbol_exists(TP_STATUS_CSUM_VALID linux/if_packet.h HAVE_TP_STATUS_CSUM_VALID)
+    if (NOT HAVE_TP_STATUS_CSUM_VALID)
+        message(STATUS "Checksum offloading to the kernel might not be fully supported.")
+    endif ()
+
+    zeek_add_plugin(
+        Zeek AF_Packet SOURCES src/Plugin.cc src/AF_Packet.cc src/RX_Ring.cc
+        BIFS src/af_packet.bif DIST_FILES zeekctl/af_packet.py SCRIPTS scripts/init.zeek)
```

Comment:
Hmm, zeekctl already ships af_packet packet since a while. Seems reasonable t o drop it from the main-tree.